EasyLearn/app/views/shared/home/_cards_classes.html.erb

```diff
@@ -0,0 +1,68 @@
+      <div class="card-body">
+        <h5 class="card-title">Implemented AJAX with PHP</h5>
+        <p class="card-text">Learn how to share information from AJAX to PHP and viceversa.</p>
+        <%= link_to "Take calss", "#", class: "btn" %>
+      </div>
+      <div class="card-footer">
+        <small class="text-muted">
+          <span class="material-icons icon-profile">account_circle</span>
+          <span class="owner-name">Mauricio Navarro</span>
+        </small>
+      </div>
+    </div>
+    
+    <div class="card">
+      <iframe src="https://www.youtube.com/embed/GfCUnf2y4lY" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+      <div class="card-body">
+        <h5 class="card-title">Contigous allocation - Calloc</h5>
+        <p class="card-text">
+          Calloc, is another function that we can use for dynamic memory management, this works in a similar way to malloc.
+        </p>
+        <%= link_to "Take calss", "#", class: "btn" %>
+      </div>
+      <div class="card-footer">
+        <small class="text-muted">
+          <span class="material-icons icon-profile">account_circle</span>
+          <span class="owner-name">Josue Esau</span>
+        </small>
+      </div>
+    </div>
+    
+    <div class="card">
+      <iframe src="https://www.youtube.com/embed/fmyvWz5TUWg" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+      <div class="card-body">
+        <h5 class="card-title">Rails - full course</h5>
+        <p class="card-text">Learn Rails from 0 in 4 hours.</p>
+        <%= link_to "Take calss", "#", class: "btn" %>
+      </div>
+      <div class="card-footer">
+        <small class="text-muted">
+          <span class="material-icons icon-profile">account_circle</span>
+          <span class="owner-name">Francisco</span>
+        </small>
+      </div>
+    </div>
+   
+    <div class="card">
+      <iframe src="https://www.youtube.com/embed/RqQ1d1qEWlE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+      <div class="card-body">
+        <h5 class="card-title">Javascript</h5>
+        <p class="card-text">Learn JavaScript from 0.</p>
+        <%= link_to "Take calss", "#", class: "btn" %>
+      </div>
+      <div class="card-footer">
+        <small class="text-muted">
+          <span class="material-icons icon-profile">account_circle</span>
+          <span class="owner-name">Alan</span>
+        </small>
+      </div>
+    </div>
+
```

Comment:
When there are repeating elements that have the same structure and only the text changes, you can make use of the [internationalization](https://guides.rubyonrails.org/i18n.html) property to simplify your code.

Reply:
I read some about this property, but I think that is something more advanced for us, but thanks for the advice, we work on it to try to solve this inconvenient.